### PR TITLE
Diego flores

### PR DESCRIPTION
**Pull request by Dfloresa**

### CODE REVIEW

File: DiegoFlores/if-else.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+public class Solution {
+
+
+
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        int number = scanner.nextInt();
+        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
```
Comment (edbermeo) [inline DiegoFlores/if-else.java:11]: Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings

File: DiegoFlores/if-else.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+        if(number%2 == 1 || ( number > 6 && number <= 20 )){
+            System.out.println("Weird");
```
Comment (edbermeo) [inline DiegoFlores/if-else.java:14]: Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings

File: DiegoFlores/if-else.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        }else if( number > 2 && number < 5  || (number > 20)){
+            System.out.println("Not Weird");
```
Comment (edbermeo) [inline DiegoFlores/if-else.java:16]: Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings

**Review by edbermeo — CHANGES_REQUESTED** [review]

**Review by Dfloresa — COMMENTED** [review]
Magic strings Fixes

**Review by edbermeo — APPROVED** [review]

**Review by luisalanmz — APPROVED** [review]
LTGM

**Pull request merged**